Mock/getMock.js

```diff
@@ -1,6 +1,6 @@
 export const getMock = {
   success: {
-    path: 'tests/get/users?id=J8NR45UzDffyMY0wBNoa',
+    path: 'tests/get/users/J8NR45UzDffyMY0wBNoa',
```

Comment:
why do we have such complicated ids? I know sometimes it's useful to prevent users from adding some random id and checking info but I think it would be a good idea to configure if we want simple id's or complex ones. @LucasZibell can we do this in a later version?

Reply:
is this the path you are going to send in the `firestoreService.GET(...)` method? if so, why is this included in it: `tests/get`

Reply:
We have those id's because they were automatically generated by fb. Yes we can, and we will do it in a future version. I was planning to change that when we release a setup file to run the tests in any db. 
I didn't understand the second question 🤔

Reply:
the second question is: this mock sets the path to: `'tests/get/users/J8NR45UzDffyMY0wBNoa'` is that path the one that goes inside the GET call in the test (like this: `firestoreService.GET('tests/get/users/J8NR45UzDffyMY0wBNoa')`) if so why do we have the `tests/get` there? shouldn't it be `firestoreService.GET('users/J8NR45UzDffyMY0wBNoa')`?

Reply:
This is the route we currently have in the db. `tests/get/users/J8NR45UzDffyMY0wBNoa`